app/src/main/java/com/lastaoutdoor/lasta/ui/screen/social/components/FriendsList.kt

```diff
@@ -110,51 +111,42 @@ fun FriendsCard(friend: UserModel, navToFriend: () -> Unit) {
                 verticalAlignment = Alignment.CenterVertically,

```

Comment:
This row is inside a column layout with padding of 8.dp. The column layout here is not necessary, instead just apply the padding to the row and remove the outsider column!